main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"github.com/gin-gonic/gin"
+	"net/http"
+	"tasks-dispatcher/api/controllers/v1"
+)
+
+func main() {
+	 router := gin.Default()
```

Comment:
I think it's better to move the route code to another package

Reply:
It makes sense. I was going to contrive the project structure following the [guideline](https://github.com/golang-standards/project-layout). I'll append this change although I thought I would create a new issue to re-design before your suggestions.